content/intro-to-writing-data.md

```diff
@@ -161,26 +161,38 @@ We’re going to create a script to ping an on-chain program that increments a c
 We'll start by using the same packages and `.env` file we made earlier in [intro to cryptography](./intro-to-cryptography.md):
 
 ```typescript
-import { Keypair } from "@solana/web3.js";
-import * as dotenv from "dotenv";
-import base58 from "bs58";
+import web3 from "@solana/web3.js";
+import dotenv from "dotenv";
 import { getKeypairFromEnvironment } from "@solana-developers/node-helpers"
 
 dotenv.config();
 
-const payer = getKeypairFromEnvironment('SECRET_KEY')
-const connection = new web3.Connection(web3.clusterApiUrl('devnet'))
+function initializeKeypair(): web3.Keypair {
+  const initPayer = getKeypairFromEnvironment("SECRET_KEY");
+  console.log(` ✅ Loaded payer keypair ${initPayer.publicKey.toBase58()}`);
+  return initPayer;
+}
 
 ```
 
-### 4. Ping program
+### 2. Create a connection
 
 Now that we've loaded our keypair, we need to connect to Solana’s Devnet. Let's create a connection:
 
 ```typescript
-const connection = new web3.Connection(web3.clusterApiUrl('devnet'))
+async function main() {
```

Comment:
This (on the left) is the step to create a connection. 

We now use esrun which supports top-level await, meaning that the `main()` function wrapper isn't needed.